src/main/java/frc/robot/KeyboardBindings.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import frc.utils.controllers.keyboard.KeyboardController;
+
+public class KeyboardBindings {
+
+    private static final KeyboardController KEYBOARD_CONTROLLER = new KeyboardController();
+
+    public static void configurateBindings() {
```

Comment:
pick one name to configurateBindings (in joystick its configure) and make it in all